packages/react-server-cli/gulpfile.babel.js

```diff
@@ -0,0 +1,8 @@
+var gulp = require("gulp"),
+	babel = require("gulp-babel");
+
+gulp.task("default", () => {
+	return gulp.src("src/**/*.js")
+		.pipe(babel({presets: ["es2015", "react"], plugins: ["transform-runtime"]}))
+		.pipe(gulp.dest("target"));
+});
```

Comment:
Thanks; should be fixed in b48525e. (I did not try to factor out the eslint task, though...)